src/service/viewer-impl.js

```diff
@@ -67,19 +67,27 @@ export const ViewportType = {
 /**
  * Visibility state of the AMP document.
  * @enum {string}
- * @private
  */
 export const VisibilityState = {
-
   /**
-   * Viewer has shown the AMP document.
+   * The AMP document is currently active and visible.
    */
   VISIBLE: 'visible',
 
   /**
-   * Viewer has indicated that AMP document is hidden.
+   * The AMP document is active but the browser tab is not.
+   */
+  HIDDEN: 'hidden',
```

Comment:
And, of course, part of this work, we need to send Eric a list of requirements for what visiblityState he should be sending us and when.